src/Splat.SimpleInjector/README.md

```diff
@@ -0,0 +1,36 @@
+# Splat.SimpleInjector
+
+## Using SimpleInjector
+
+Splat.SimpleInjector is an adapter for `IMutableDependencyResolver`.  It allows you to register your application dependencies in a SimpleInjector `Container`.  You can then use the container as Splat's internal dependency resolver.
+
+### Register the Container
+
+Because of internal SimpleInjector behaviours in order to register it as a container intermediate "dummy" container has to be used - `SimpleInjectorInitializer`.
+
+```cs
+SimpleInjectorInitializer initializer = new SimpleInjectorInitializer();
+Locator.SetLocator(initializer);
+
+// Register ReactiveUI dependencies
```

Comment:
are these 2 initialize lines actually needed? Splat and ReactiveUI would do these during normal operation. Is there a nuance we need to document so people are aware?